django/chat/templates/chat/components/chat_options_sidebar.html

```diff
@@ -7,7 +7,8 @@
   <div class="border-start h-100 position-relative">
     <button id="right-sidebar-resize-handle" class="btn btn-none" type="button">
       <i class="bi bi-grip-vertical"></i>
-      <span class="visually-hidden">{% trans "Drag or use the left and right arrows to resize." %}</span>
+      <span class="visually-hidden">{% trans "Drag or use the left and right arrows to
+      resize." %}</span>
```

Comment:
Did something go wrong with DjLint? If these lines are split the translation tag will not work.